package.json

```diff
@@ -44,9 +45,10 @@
     "grunt-qunit-tap": "~2.0.0",
     "grunt-string-replace": "~0.2.4",
     "grunt-sync": "^0.6.2",
-    "karma": "^1.3.0",
+    "karma": "^4.4.1",
```

Comment:
are you sure we need update karma version?

Reply:
I've restored the "old" version of Karma